db/migrate/20200728054557_refer_config_proxy_to_http_proxy.rb

```diff
@@ -0,0 +1,56 @@
+class ReferConfigProxyToHttpProxy < ActiveRecord::Migration[6.0]
+  def up
+    add_column :foreman_virt_who_configure_configs, :http_proxy_id, :integer
+
+    User.as_anonymous_admin do
```

Comment:
I'm fairly certain we need to check in the anonymous admin exists. In a new installation that user won't be there and this is probably going to raise an error on the first db:migrate

`if User.where(login: User::ANONYMOUS_ADMIN).exists?` ...

Reply:
I just want to make sure that the code that i have written in this block would run. If I will use if condition and ANONYMOUS_ADMIN does not exist it will delete the proxy column without referring to http_proxy_id. I do not want to loose any data. Please suggest some other way to do this.

Reply:
In a fresh installation there is no data to migrate so that'll be OK

Reply:
Just to be clear we still need to handle the data migration in general in the case where the system is upgraded from an older version at which point this migration will run. :)